app/src/main/java/server/RouteList.java

```diff
@@ -0,0 +1,54 @@
+package server;
+
+import java.util.*;
+
+public class RouteList {
+    HashMap<String, Route> startupList;
+
+    public RouteList() {
+        this.startupList = constructRouteList();
+    }
+
+    private HashMap<String, Route> constructRouteList() {
```

Comment:
Another little naming thing, but I feel like `RouteList` is a little misleading. I would expect it to return a `List` but it returns a `HashMap`. @davetorre Do you have any thoughts on this? Would something like `RouteMap` be clearer?